app/src/main/java/com/infomaniak/swisstransfer/ui/theme/Dimens.kt

```diff
@@ -20,13 +20,15 @@ package com.infomaniak.swisstransfer.ui.theme
 import androidx.compose.ui.unit.dp
 
 object Dimens {
+    val MaxSinglePaneScreenWidth = 800.dp
     val SettingHorizontalMargin = Margin.Medium
     val SettingVerticalMargin = Margin.Small
     val DescriptionWidth = 300.dp
     val LargeButtonHeight = 56.dp
-    val DoubleButtonMaxWidth = 800.dp
+    val DoubleButtonMaxWidth = MaxSinglePaneScreenWidth
     val SingleButtonMaxWidth = DoubleButtonMaxWidth / 2
     val SmallIconSize = 16.dp
     val IconSize = 24.dp
     val BorderWidth = 1.dp
+    val ButtonComboVerticalPadding = Margin.Small
```

Comment:
The same way, I'd not put the margins directly inside `DoubleButtonCombo` so they are not forced everywhere this component is used, I also wouldn't have a dimen that's called `ButtonComboVerticalPadding` to not force it everytime a `DoubleButtonCombo` is used. Else this would directly be defined inside `DoubleButtonCombo`.

Wouldn't `ScaffoldButtonComboVerticalPadding` be better, this way it limits the scope to only scaffolds but not like all of the possible usages?